#!/usr/bin/env python
PACKAGE = "vmxpi_ros_cam"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("iLowH", int_t, 0, "Lower bound Hue", 6, 0, 179)
gen.add("iHighH", int_t, 0, "Upper bound Hue", 87, 0, 179)
gen.add("iLowS", int_t, 0, "Lower bound Saturation", 104, 0, 255)
gen.add("iHighS", int_t, 0, "Upper bound Saturation", 255, 0, 255)
gen.add("iLowV", int_t, 0, "Lower bound Value", 178, 0, 255)
gen.add("iHighV", int_t, 0, "Upper bound Value", 255, 0, 255)

gen.add("tracking_on", bool_t, 0, "Enable bounding rectangle", True)

exit(gen.generate(PACKAGE, "opencv_settings", "OpenCVSettings"))